architecture/3-DataStructures/2-ComponentListItem.yaml

```diff
@@ -56,4 +57,9 @@ elements:
     itemType: ''
     description: Developer comments on this part or tool.
     derivedFrom: []
+  - key: interfaces
+    type: list
+    itemType: 3-DataStructures/6-InterfaceListItem.yaml'
+    description: Listing the interfaces of this part or tool.
```

Comment:
I would reword `Listing the interfaces of this part or tool.` to `Listing of interfaces for the part or tool.`

Reply:
Resolved [4cbcc20](https://github.com/ODU-CGA-CubeSat/dof-cubesat/pull/2/commits/4cbcc2047b672037edb01d55d017f9938b041063)